cfme/test_requirements.py

```diff
@@ -10,61 +10,389 @@
-sysprep = pytest.mark.requirement("sysprep")
-tag = pytest.mark.requirement("tag")
-timelines = pytest.mark.requirement("timelines")
-upgrade = pytest.mark.requirement("upgrade")
-vm_migrate = pytest.mark.requirement("vm_migrate")
-vmrc = pytest.mark.requirement("vmrc")
+ansible = pytest.mark.requirement(
+    "Ansible Automation",
+    description='Ansible Automation Requirement',
+    assignee_id='sbulage',
+)
+
+access = pytest.mark.requirement(
+    "Access",
+    description='Direct integration of SaaS offerings in MIQ/CFME',
+    assignee_id='sbulage',
+)
+
+alert = pytest.mark.requirement(
+    "Alerts",
+    description='Alerting in MIQ',
+    assignee_id='jdupuy',
+)
+
+auth = pytest.mark.requirement(
+    "Authentication",
+    description='Authentication in MIQ/CFME, external authentication provider support',
+    assignee_id='apagac',
+)
+
+automate = pytest.mark.requirement(
+    "Automate",
+    description='Automate and related control behavior',
+    assignee_id='ghubale',
+)
+
+azure = pytest.mark.requirement(
+    'Azure Integration',
+    description='Integration of Azure in MIQ/CFME',
+    assignee_id='anikior',
+)
+
+bottleneck = pytest.mark.requirement(
+    "Optimization and Bottleneck",
+    description='Optimizations and Bottleneck ID, planned deprecation',
+    assignee_id='nachandr',
+)
+
+c_and_u = pytest.mark.requirement(
+    "Capacity and Utilization",
+    description='Capacity and Utilization data collection',
+    assignee_id='nachandr',
+)
+
+multi_tenancy = pytest.mark.requirement(
+    "MIQ Multi-Tenancy",
+    description='Support for Tenants in MIQ/CFME',
+    assignee_id='mnadeem',
+)
+
+chargeback = pytest.mark.requirement(
+    "Chargeback",
+    description='Chargeback rates, calculations, and reports',
+    assignee_id='tpapaioa',
+)
+
+cloud_init = pytest.mark.requirement(
+    "Provisioning with cloud_init",
+    description='Provisioning lifecycle with cloud-init resource configuration',
+    assignee_id='',  # FIXME needs assigned
+)
+
+configuration = pytest.mark.requirement(
+    "Configuration",
+    description='Configuration pages of MIQ/CFME, also partially covered by specific requirements',
+    assignee_id='tpapaioa',
+)
+
+control = pytest.mark.requirement(
+    "Control",
+    description='Control / Compliance policies and enforcement',
+    assignee_id='mmojzis',
+)
+
+dashboard = pytest.mark.requirement(
+    "Dashboard",
+    description='MIQ/CFME Dashboards creation, usability',
+    assignee_id='jhenner',
+)
+
+dialog = pytest.mark.requirement(
+    "Service Dialogs",
+    description='Service dialogs, creation and form behavior on order',
+    assignee_id='nansari',
+)
+
+discovery = pytest.mark.requirement(
+    "Provider Discovery",
+    description='Discovery of Cloud/Infra/etc providers',
+    assignee_id='ndhandre',
+)
+
+distributed = pytest.mark.requirement(
+    "Distributed Appliances",
+    description='Distributed Appliance deployment, configuration, and interaction',
```

Comment:
@mshriver, isn't this mine ?

Reply:
I went with what assignments were on the page when I opened the branch, they're shifting for 5.11. I'd prefer to not jumble around all the assignee's right now, and let contributors update this field as they hand-off responsibility.